inst/include/temperature_component.hpp

```diff
@@ -72,27 +72,27 @@ class TemperatureComponent : public IModelComponent {
     void setoutputs(int tstep);
 
     // Hard-coded DOECLIM parameters
-    const int dt = 1;                     // years per timestep (this is implicit in Hector)
+    double dt = 1;                     // years per timestep (this is implicit in Hector)
```

Comment:
I changed it because in the `temperature_component.cpp` any time dt was called it appeared as `double(dt)` which seemed redundant I figured that if dt was created as a double then we could avoid all of those extra calls. 

Should I left all the double calls in the `temperature_component.cpp`, I was just trying to streamline the code/make it easier to read. As a new a cpp coder I am not sure if that was the right move or not so feedback welcome. 

Also Hector complies and runs with `const in dt = 1` without the all the double calls in the `temperature_component.cpp`  is that the correct move?

Reply:
The change to a double (so eliminating all those `double(dt)` usages) is fine; I'm just asking why not declare this as `const double dt = 1;`.

Reply:
That's a great idea!